tests/unit/anchore_engine/clients/test_grype_wrapper.py

```diff
@@ -100,12 +101,53 @@ def test_get_missing_grype_db_session():
         # Function under test
         grype_wrapper_singleton._grype_db_session_maker
 
+    # Validate error message
     assert (
         str(error.value)
         is GrypeWrapperSingleton.MISSING_GRYPE_DB_SESSION_MAKER_ERROR_MESSAGE
     )
 
 
+def test_get_current_grype_db_checksum(grype_db_dir):
+    # Create grype_wrapper_singleton instance
+    grype_wrapper_singleton = TestGrypeWrapperSingleton.get_instance()
+
+    # Setup test input
+    grype_wrapper_singleton._grype_db_dir = grype_db_dir
+
+    # Function under test
+    result = grype_wrapper_singleton.get_current_grype_db_checksum()
+
+    # Validate result
+    assert result == NEW_VERSION_NAME
+
+
+def test_get_current_grype_db_checksum_missing_db_dir_value():
+    # Create grype_wrapper_singleton instance
+    grype_wrapper_singleton = TestGrypeWrapperSingleton.get_instance()
+
+    # Setup test input with a non-existant path
+    grype_wrapper_singleton._grype_db_dir = "/does/not/exist"
+
+    # Function under test
+    result = grype_wrapper_singleton.get_current_grype_db_checksum()
+
+    # Validate error message
+    assert result is None
```

Comment:
should this be an exception case too?

Reply:
No. The exception is raised if the varliable is not set. The current (and soon-to-be-replaced) logic only returns None if the variable is set but the dir for some reason doesn't actually exist. Should never happen.

Reply:
Ah I see the match parameter in raises() now, I didn't know about that one. Thanks!